packages/cactus-plugin-ledger-connector-iroha/package.json

```diff
@@ -60,12 +60,13 @@
     "@types/google-protobuf": "3.15.5",
     "axios": "0.21.4",
     "express": "4.17.1",
+    "fast-safe-stringify": "2.1.1",
     "grpc": "1.24.11",
+    "iroha-helpers": "^1.3.0",
     "iroha-helpers-ts": "0.9.25-ss",
```

Comment:
Isn't conflict between those two? Shouldn't one of them be removed?

Reply:
Yes, I will remove "iroha-helpers-ts".

Reply:
@yashrajdesai: +1 to what @baziorek said!